### PR TITLE
Require requests-toolbelt v1+ for compatiblity with urllib3

### PR DESCRIPTION
**Pull request by hugovk**
# Pull Request Check List

Resolves: #8679.

- [ ] Added **tests** for changed code.
- [ ] Updated **documentation** for changed code.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -50,7 +50,7 @@ pkginfo = "^1.9.4"
 platformdirs = "^3.0.0"
 pyproject-hooks = "^1.0.0"
 requests = "^2.26"
-requests-toolbelt = ">=0.9.1,<2"
+requests-toolbelt = ">=1,<2"
```
Comment (Secrus) [inline pyproject.toml:53]: Could you use the `^1.0.0` marker, please? It will mean the same in a nicer way.

Comment (hugovk) [reply pyproject.toml]: Sure, updated!

**Review by Secrus — APPROVED** [review]

**Comment by hugovk** [comment]
Unrelated: is it time to add 3.12 as a required check?

image

**Comment by Secrus** [comment]
> Unrelated: is it time to add 3.12 as a required check?
> 
> image

Yeah, it is supposed to be required. I poked the people with admin rights to fix this ;)

**Pull request merged**